.github/workflows/driver_ci.yaml

```diff
@@ -0,0 +1,135 @@
+        run: |
+          ./scripts/run_static_code_analysis.sh
+
+  unit_testing_coverage:
+    runs-on: ubuntu-latest
+    name: "Unit testing + coverage ${{ matrix.csiblock-component }}"
+    strategy:
+      matrix:
+       include:
+         - csiblock-component: controller
+           command-to-run: "./scripts/run_unitests.sh"
+         - csiblock-component: node
+           command-to-run: "make test-xunit-in-container"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: CSI-${{ matrix.csiblock-component }}- Unit testing + coverage
+        run: |
+          mkdir -p build/reports && chmod 777 build/reports
+          $command_to_run `pwd`/build/reports
+        env:
+          command_to_run: ${{ matrix.command-to-run }}
+
+  k8s_yamls_validation:
+    runs-on: ubuntu-latest
+    name: "Deployment k8s yamls validation"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: CSI-deployment- k8s yamls validation
+        run: |
+          ./scripts/run_yamlcheck.sh
+
+  csi_build_and_push_images:
+    runs-on: ubuntu-latest
+    name: "Build and push images"
+    needs:
+      - CSI_controller_static_code_analysis
+      - Unit_testing_coverage
+      - k8s_yamls_validation
+      - Prepare_env
+    strategy:
+      matrix:
+        image_type: ['node', 'controller']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1.3.0
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: '${{ secrets.CSIBLOCK_DOCKERHUB_USERNAME }}'
+          password: '${{ secrets.CSIBLOCK_DOCKERHUB_PASSWORD }}'
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-${{ matrix.image_type }}
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Build and push ${{ matrix.image_type }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/ppc64le,linux/s390x
+          push: true
+          tags: |
+            ${{ secrets.CSIBLOCK_DOCKERHUB_USERNAME }}/ibm-block-csi-${{ matrix.image_type }}:${{ needs.Prepare_env.outputs.driver_images_specific_tag }}
+            ${{ secrets.CSIBLOCK_DOCKERHUB_USERNAME }}/ibm-block-csi-${{ matrix.image_type }}:${{ needs.Prepare_env.outputs.docker_image_branch_tag }}
+          file: Dockerfile-csi-${{ matrix.image_type }}
+          cache-from: type=local,src=/tmp/.buildx-${{ matrix.image_type }}
+          cache-to: type=local,dest=/tmp/.buildx-new-${{ matrix.image_type }}
+        # Temp fix
+        # CSI-3164
+        # https://github.com/docker/build-push-action/issues/252
```

Comment:
could you check if this [is still needed](https://github.com/docker/build-push-action/issues/252#issuecomment-889357925)?